packages/p2p/src/components/buy-sell/popup.jsx

```diff
@@ -0,0 +1,207 @@
+import React, { Fragment, Component }      from 'react';
+import PropTypes                           from 'prop-types';
+import { Formik, Field, Form }             from 'formik';
+import { Input, Button, ThemedScrollbars } from 'deriv-components';
+import { localize, Localize }              from 'deriv-translations';
+import IconBack                            from '../../assets/icon-back.jsx';
+import IconClose                           from '../../assets/icon-close.jsx';
+
+class Popup extends Component {
+
+    handleSubmit = (formik_vars, { setSubmitting }) => {
+        // TODO: p2p API call to create ad
+        // eslint-disable-next-line no-console
+        console.log(this.state);
+        // eslint-disable-next-line no-console
+        console.log(formik_vars);
+        setSubmitting(false);
+    }
+
+    getInitialValues = () => {
+        const { ad } = this.props;
+        const is_buy = ad.type === 'buy';
+        const amount_currency = ad.min_transaction;
+        const amount_asset = ad.min_transaction / ad.fix_price;
+        const initial_send = is_buy ? amount_currency : amount_asset;
+        const initial_receive = is_buy ? amount_asset : amount_currency;
+        const send_currency = is_buy ? ad.currency : ad.asset;
+        const receive_currency = is_buy ? ad.asset : ad.currency;
+        const send_validation = is_buy ? amount_currency : amount_asset;
+        const receive_validation = is_buy ? amount_asset : amount_currency;
```

Comment:
one `if` is easier to deal with in the long run then a `ternary` for each I think
```
if (is_buy) {
  return ...
else {
  return ...
}
```